src/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  HashRouter, Routes, Route, Navigate,
+} from 'react-router-dom';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { App } from './App';
+import { PageNotFound } from './components/PageNotFound';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people">
+          <Route path=":slug?" element={<PeoplePage />} />
+        </Route>
+
+        <Route path="home" element={<Navigate to="../.." />} />
```

Comment:
```suggestion
        <Route path="home" element={<Navigate  to="/" replace />} />
```
It looks a bit strange